src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasBySample.java

```diff
@@ -52,7 +53,7 @@
  */
 @DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Number of forward and reverse reads that support REF and ALT alleles (SB)")
 public final class StrandBiasBySample extends GenotypeAnnotation implements StandardMutectAnnotation {
-    private final static Logger logger = LogManager.getLogger(StrandBiasBySample.class);
+    private final static OneShotLogger logger = new OneShotLogger(LogManager.getLogger(StrandBiasBySample.class));
```

Comment:
Here too I think the `OneShotLogger` should be separate from the normal logger, with a clear name like `oneShotLogger`, so that people don't mistakenly try to write additional messages to the `OneShotLogger` and wonder why they don't show up.

Reply:
FWIW in the situation I'm trying to untangle now, having this be repeated is actually useful.  Since there are (I think) reasonable cases where strand bias can't be computed, but in my case the fact that it came out ~350 times for a fairly small panel was a good indicator of error.  And we're not looking back through historic logs to find other samples/runs that have high rates of this log message.